packages/pwa-kit-extension-sdk/src/configs/webpack/overrides-resolver-plugin.ts

```diff
@@ -82,7 +82,13 @@ export class OverridesResolverPlugin {
                 ...this.options.resolveOptions
             })
         } catch (e: any) {
-            return callback(e)
+            // Return a better error message
+            return callback(
+                new Error(
+                    // sourcePath, in this case, points to a more specific file that is more helpful for debugging
+                    `Cannot find module '${importPath}' from '${sourcePath}'`
+                )
+            )
```

Comment:
Bonus: if webpack cannot resolve the import path with the override syntax, then we'll return an error mesage that is clearer. Previously, it would look as if the error comes from the base project. Now it looks like this:

```
ModuleNotFoundError: Module not found: Error: Cannot find module '$/components/with-red-borderrrrrrrrrrr' from '/Users/vmarta/repos/pwa-kit/packages/template-typescript-minimal/node_modules/@salesforce/extension-sample-2/src/components/with-using-foo.tsx'

```